scripts/get_stat.cmd

```diff
@@ -0,0 +1,27 @@
+@setlocal enabledelayedexpansion
+@echo off
+
+rem  usage: test_stat.cmd <sleepy_file_basename> <stat_descr> <target_envvar>
+rem  Get <stat_descr> from Stats.txt and save its value in <target_envvar>
```

Comment:
GitHub doesn't like these, and looking here:

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/rem

> You can't use a redirection character (`<` or `>`) or pipe (`|`) in a batch file comment.

seems to imply there might be a problem.

Reply:
I don't mind conforming to the docs, but it my experience, this has never caused a problem.

On my Win10 system, running a batch file with this produces no output files, errors, I can pipe text into it, redirect stdout/stdin without issue.

```
REM  usage: t1 <t1_arg1>
REM         t1 | t2_pipe
REM  usage  t1 <t1_in
REM  usage  t1 >t1_out

echo ABCD EFGH

set /p myvar=What is your name?
echo Hello, %myvar%
```

I'm curious about the circumstances under which including these chars in a `REM` comment would do something. Has this ever failed for you?

Reply:
Maybe it's some legacy from some old DOS version when `rem` wasn't a builtin.